src/Wizard.jl

```diff
@@ -1145,25 +1139,353 @@ function step7(state::WizardState)
       email: false
     git:
       depth: 99999999
-    sudo: required
-    services:
-      - docker
-
+    cache:
+      timeout: 1000
+      directories:
+        - downloads
     env:
-      DOCKER_IMAGE: staticfloat/julia_crossbuild:x64
+      global:
+        - BINARYBUILDER_DOWNLOADS_CACHE=downloads
+    sudo: required
 
     # Before anything else, get the latest versions of things
     before_script:
-      - docker pull \$DOCKER_IMAGE
       - julia -e 'Pkg.clone("https://github.com/JuliaPackaging/BinaryProvider.jl")'
       - julia -e 'Pkg.clone("https://github.com/JuliaPackaging/BinaryBuilder.jl"); Pkg.build()'
 
     script:
       - julia build_tarballs.jl
-    ```
+      
+    before_cache:
+      - rm -rf \$(julia -e 'println(Pkg.dir("BinaryBuilder","deps","root"))')
+    """)
+end
+
+function print_travis_deploy(io, repo, secure_key)
+    print(io,
+    """
+    deploy:
+        provider: releases
+        api_key:
+            # Note; this api_key is only valid for $(GitHub.name(repo)); you need
+            # to make your own: https://docs.travis-ci.com/user/deployment/releases/
+            secure: $secure_key
+        file_glob: true
+        file: products/*.tar.gz
+        skip_cleanup: true
+        on:
+            repo: $(GitHub.name(repo))
+            tags: true
     """)
 end
 
+mutable struct termios
+	c_iflag::Cint
+	c_oflag::Cint
+	c_cflag::Cint
+	c_lflag::Cint
+	c_line::UInt8
+	c_cc::NTuple{19, UInt8}
```

Comment:
Weird indentation here. (Tabs on these lines and 4 space on the next and everywhere else)